### PR TITLE
Update docs regarding expired tokens

### PR DESCRIPTION
**Pull request by wainersm**
So we record what should be done next time :)

### CODE REVIEW

**Review by fitzthum — APPROVED** [review]
Nice

**Review by bpradipt — APPROVED** [review]
/lgtm

File: github/azure-self-hosted-runners/README.md [hunk 1]
```diff
@@ -22,19 +22,15 @@ The application is deployed as an [ACI](https://azure.microsoft.com/en-us/produc
 
 ### Deployment
 
-Github tokens (see above) need to be passed to Garm via tf variables, either by creating a `tf/terraform.tfvars` or specifying it on the cli:
+Github tokens (see above) are passed to Garm via Azure Key Vault secrets. So first you should put the tokens as secrets in
+an Key Vault. The Key Vault id is then passed to terraform either by creating a `tf/terraform.tfvars` or specifying it on the cli:
```
Comment (bpradipt) [inline github/azure-self-hosted-runners/README.md:26]: nit: s/an/a

**Review by mkulke — APPROVED** [review]
thank you!

File: github/azure-self-hosted-runners/README.md (hunk 1 above)
Comment (wainersm) [reply github/azure-self-hosted-runners/README.md]: done

**Pull request merged**